lib/rails_cursor_pagination/paginator.rb

```diff
@@ -443,14 +477,14 @@ def id_column
     # So our SQL WHERE clause needs to be something like:
     #    WHERE author > 'Jane' OR author = 'Jane' AND id > 4
     #
-    # @return [ActiveRecord::Relation]
+    # @return [ActiveRecord::Relation, ActiveRecord::QueryMethods::WhereChain]
     def filtered_and_sorted_relation
       memoize :filtered_and_sorted_relation do
         next sorted_relation if @cursor.blank?
 
         unless custom_order_field?
           next sorted_relation.where "#{id_column} #{filter_operator} ?",
```

Comment:
Given that `#id_column` prefixes the column with the table name and also escapes it with backticks ``` ` ```, does this even work for computed columns, like `MAX(id)`? 🤔  I would think that the DB would complain that it does not know about a `my_table.MAX(id)` column..

Reply:
It won't work with just `MAX(id)` unless that is also aliased to `id` (`SELECT MAX(id) AS id, ...`) I think. We could do the same as in the order method where the column is left as it is if it's a string.

Reply:
Yes, that would then probably be best.. We'll just have to make sure that it works consistently.